data/monitoring/prom-rule.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus.cnao.io: ""
+  name: prometheus-rules-cluster-network-addons-operator
+  namespace: {{ .Namespace }}
+spec:
+  groups:
+    - name: kubevirt.cnao.rules
+      rules:
+        - expr: sum(up{namespace='{{ .Namespace }}', pod=~'cluster-network-addons-operator-.*'} or vector(0))
+          record: kubevirt_cnao_num_up_operators
+        - alert: CnaoDown
+          annotations:
+            summary: CNAO pod is down.
+          expr: kubevirt_cnao_num_up_operators == 0
+          for: 5m
```

Comment:
aren't 5 minutes is too much ?
one might want to know about the alert faster

Reply:
I don't think there is cause to get it faster.
This is aligned with kubevirt's similar alert [VirtAPIDown](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-operator/resource/generate/components/crds.go#L564-L572)

Reply:
Please check https://github.com/kubevirt/kubevirt/issues/5383 and ensure that this alert works.

Reply:
@sradco Do we have a naming convention for alerts? Should it start with `Kubevirt`? See https://github.com/kubevirt/monitoring/tree/main/runbooks

Reply:
@RamLavi No naming convention, but you are missing runbook_url and a PR for the runbook in kubevirt/monitoring

Reply:
> Please check [kubevirt/kubevirt#5383](https://github.com/kubevirt/kubevirt/issues/5383) and ensure that this alert works.

I'm not using `labeldrop` so it shouldn't occur in this case. Also, I checked this on the GUI using CRC (see images in the PR description)

>@RamLavi No naming convention, but you are missing runbook_url and a PR for the runbook in kubevirt/monitoring

You are right. I will issue the relevant PRs for runbooks soon.

Reply:
@RamLavi The runbook_url should be part of this PR.

Reply:
Example of runbook_url https://github.com/kubevirt/hyperconverged-cluster-operator/pull/1448/files